layers/+lang/java/packages.el

```diff
@@ -462,7 +462,12 @@
         "rem" 'lsp-java-extract-method
         "cc"  'lsp-java-build-project
         "an"  'lsp-java-actionable-notifications
-        "="   'lsp-format-buffer)
+        "="   'lsp-format-buffer
+
+        ;; dap-mode
+        "xj"   'dap-java-debug
```

Comment:
This `x` feels like a weird prefix. Why have you chosen this one?

Reply:
I used `x` since it is available and wont be missed in the hydra - e. g. you will be able to navigate readonly with hydra on. Also it comes from `eXecute`.

Reply:
I think this should be under the `debug` prefix as well. Unless there is a use-case for it being outside that I'm missing.